android/fastlane/Fastfile

```diff
@@ -37,8 +37,10 @@ desc "Create an Android build in release mode. Set the environment variable E2E_
 lane :build do |options|
   version_code = options[:version_code]
   version_name = options[:version_name]
+  build_config_name = options[:build_config_name]
 
   ENV["BUNDLE_CONFIG"] = "./metro.config.ci.js"
+  ENV["BUILD_CONFIG_NAME"] = build_config_name
```

Comment:
I'm not sure but maybe it makes sense not to pass this as environment variable but as Gradle Property. See below.

Reply:
```
properties: {
          :VERSION_CODE => version_code,
          :BUILD_CONFIG_NAME => build_config_name,
          :KEYSTORE_PATH => KEYSTORE_PATH,
      }
```

Reply:
Okay, yes, I like the idea and didn't think of it. However, to make it possible to also specify the build config from a script in the package.json, this needs to happen via environment variables. What do you think?

Reply:
Discussed: Environment variables are necessary for the scripts for now.